pontoon/pretranslation/pretranslate.py

```diff
@@ -70,12 +74,23 @@ def get_translations(entity, locale):
 
     # Else fetch from google translate
     elif locale.google_translate_code:
+        entity_string = (
```

Comment:
This `entity_string` is different from the one above, which is confusing. They should probably have different names.